packages/preview/ssrn-scribe/0.4.8/template/main.typ

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////
+#import "@preview/ssrn-scribe:0.4.8": *
+///////////////////////////////
+
+
+#show: thmrules
+#set page(numbering: "1")
```

Comment:
This should probably be part of the template.

Reply:
added in the template file